charts/cray-psp/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: cray-psp
+version: 0.2.0
+description: Apply net-raw PSP
+keywords:
+  - cray-psp
+home: https://github.com/Cray-HPE/cray-psp
+maintainers:
+  - name: kburns-hpe
+  - name: brantk-hpe
```

Comment:
I don't think I had anything to do with the contents of this chart.

Reply:
Removed.